hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/HddsVolume.java

```diff
@@ -357,12 +357,14 @@ public void loadDbStore(boolean readOnly) throws IOException {
 
     File storageIdDir = new File(clusterIdDir, getStorageID());
     if (!storageIdDir.exists()) {
+      dbLoadFailure.set(true);
```

Comment:
Should we add this to line 354 where the `clusterIdDir` doesn't exist?

Reply:
"Nonexisting clusterIdDir" is considered as NOT_FORMATTED, and the who structure is recreated when loading.